src/tools/ci/docker/linux_fedora_rawhide

```diff
@@ -0,0 +1,46 @@
+FROM fedora:rawhide
+RUN \
+  yum -y update && \
+  yum -y install \
+  git \
+  meson \
+  perl \
+  perl-IPC-Run \
+  systemd-devel \
+  \
+  bison \
+  ccache \
+  clang \
+  flex \
+  gcc \
+  lz4 \
+  ninja-build \
+  \
+  libicu-devel \
+  libuuid-devel \
+  libxml2-devel \
+  libxslt-devel \
+  llvm-devel \
+  lz4-devel \
+  pam-devel \
+  python3-devel \
+  readline-devel \
+  tcl-devel \
+  \
+  openldap-devel \
+  openldap-servers \
+  openldap-clients \
+  \
+  openssl \
+  openssl-devel \
+  \
+  krb5-devel \
+  krb5-server \
+  krb5-server-ldap \
+  krb5-workstation \
+  \
+  langpacks-en \
+  glibc-langpack-en
+RUN \
+  find /usr/share/perl5/ExtUtils -name typemap -exec ln -sf '{}' /usr/share/perl5/vendor_perl/ExtUtils/typemap ';'
```

Comment:
Hm, what necessitates the ln bit here? It's generally a good idea to add comments for workarounds like this.